osu.Game/Overlays/MusicController.cs

```diff
@@ -60,11 +61,16 @@ public partial class MusicController : CompositeDrawable
         [Resolved]
         private IBindable<IReadOnlyList<Mod>> mods { get; set; } = null!;
 
+        public readonly BindableList<Live<BeatmapSetInfo>> Playlist = new BindableList<Live<BeatmapSetInfo>>();
+        public readonly BindableBool PlaylistHookedByOverlay = new BindableBool(false);
```

Comment:
What is this now? What is the meaning of 191699054e450037fb22ca08e2f944b8f2e927cd in general? I could be fine with the previous code but this seems actively worse?

Reply:
The previous code does not work if osu has no beatmap sets and the player adds beatmaps for the first time without an open playlist overlay.

If you tell me how to track whether Bindable has BindTarget, I could get rid of using PlaylistHookedByOverlay.